merlin/study/study.py

```diff
@@ -116,33 +117,44 @@ def __init__(
         self.dag = None
         self.load_dag()
 
+    @cached_property
+    def provenance_original_spec(self):
+        return os.path.join(self.info, "orig.yaml")
```

Comment:
Maybe prepend orig_ to the filename. orig_filename

Reply:
yeah i think this is a good idea. which do we prefer?

prepend (orig_name.yaml)
or insert near the end (name_orig.yaml)

Reply:
Whichever is easier.